themes/openy_themes/openy_rose/scss/modules/_paragraphs.scss

```diff
@@ -952,3 +952,54 @@
     }
   }
 }
+
+.paragraph--type--video-player .field-media-local-video {
+  padding: 20px;
+}
+
+.sidebar-region {
+  .video-js {
+    width: 300px !important;
```

Comment:
What the reason to use so many `!important` here and below?

Reply:
By default, the size of the video player is specified in the widget, it is required settings. _!important_ are needed to overwrite widget settings for sidebar or mobile screens and for full-screen mode.